qlib/contrib/data/highfreq_handler.py

```diff
@@ -124,14 +124,37 @@ def __init__(
         day_length=240,
     ):
         self.day_length = day_length
+
+        def check_transform_proc(proc_l):
```

Comment:
Why can't this function be used here 
https://github.com/microsoft/qlib/blob/a53f59cdf7f36d28f3998f2bb103c56d08cc2e63/qlib/contrib/data/handler.py#L11